Script/PRONTO.py

```diff
@@ -231,7 +232,123 @@ def clear_blank_line(file_in,file_out):
+					sys.exit(0)
+			if(sheet_material.cell_value(r,c) == col_birth_date):
+				ipd_birth_date_exl = sheet_material.cell_value(r+2,c)
+				try:
+					datetime_date = str(xlrd.xldate_as_datetime(ipd_birth_date_exl,0))
+					ipd_birth_year = datetime_date.split('-')[0]
+				except:
+					ipd_birth_year = "-"
+			if(sheet_material.cell_value(r,c) == col_gender and ipd_gender == ""):
+				ipd_gender = str(sheet_material.cell_value(r+2,c))
+			if(sheet_material.cell_value(r,c) == col_age):
+				ipd_age = str(sheet_material.cell_value(r+2,c))
+			if(sheet_material.cell_value(r,c) == col_consent and ipd_consent == ""):
+				ipd_consent = str(sheet_material.cell_value(r+2,c))
+				for r in range(r,(sample_info_row-2)):
+					if(ipd_consent == "0.0"):
+						ipd_consent = ""
+					if(sheet_material.cell_value(r,6) == col_requisition_hospital and requisition_hospital == ""):
+						requisition_hospital = sheet_material.cell_value(r+2,6)
+					if((sheet_material.cell_value(r+2,c) != "" or sheet_material.cell_value(r+2,c) != "-" or sheet_material.cell_value(r+2,c) != "0.0") and str(sheet_material.cell_value(r+2,c)) not in ipd_consent):
+						if(ipd_consent == ""):
+							ipd_consent = str(sheet_material.cell_value(r+2,c))
+						else:
+							ipd_consent = ipd_consent + "," + str(sheet_material.cell_value(r+2,c))
+						continue
+			if(sheet_material.cell_value(r,c) == col_material_name and ipd_material_id == ""):
+				for r in range(r,(extra_data_row-2)):
+					if(sheet_material.cell_value(r+2,9) == DNA_sampleID and sheet_material.cell_value(r+2,c) != "" and str(sheet_material.cell_value(r+2,c)) not in DNA_material_id):
+						if(DNA_material_id == ""):
+							DNA_material_id = str(sheet_material.cell_value(r+2,c))
+						else:
+							DNA_material_id = DNA_material_id + "," + str(sheet_material.cell_value(r+2,c))
+						tumor_content_nr = sheet_material.cell_value(r+2,2)
+						continue
+					if(RNA_sampleID != "" and sheet_material.cell_value(r+2,9) == RNA_sampleID and sheet_material.cell_value(r+2,c) != "" and str(sheet_material.cell_value(r+2,c)) not in RNA_material_id):
+						if(RNA_material_id == ""):
+							RNA_material_id = str(sheet_material.cell_value(r+2,c))
+						else:
+							RNA_material_id = RNA_material_id + "," + str(sheet_material.cell_value(r+2,c))
+						continue
+			if(sheet_material.cell_value(r,c) == col_extraction_hospital and extraction_hospital == ""):
+				for r in range(r,(library_pre_row-2)):
+					if(sheet_material.cell_value(r+2,8) == DNA_sampleID):
+						extraction_hospital = str(sheet_material.cell_value(r+2,c))
+						break
+			if(sheet_material.cell_value(r,c) == col_batch_nr and batch_nr == ""):
+				for r in range(r,(nrows_material-2)):
+					if(sheet_material.cell_value(r+2,0) == DNA_sampleID):
+						batch_nr = str(sheet_material.cell_value(r+2,c))
+	open_exl_material.release_resources()
+	if(ipd_consent == "0.0"):
+		ipd_consent = ""
+	if(ipd_age == "" and ipd_birth_date != ""):
```

Comment:
It seems that nothing happens with variable `ipd_birth_date` between assigning it empty string in the beginning of this function and the test checking if `ipd_birth_date` is not empty. This means that its value is always "" and the condition will always be `False` - is this a feature?